apps/backend-e2e/src/e2e/auth.cy.ts

```diff
@@ -15,140 +15,87 @@ describe('AUTHENTIFICATION TESTS', () => {
+  it('Should fail when trying to connect without any credentials 2/2.', () => {
     cy.request({
-      method: 'POST',
+      method: 'GET',
       url: '/api/auth/login',
-      body: {
-        email: 'admin@example.org',
-        password: faker.internet.password(),
-      },
+      body: {},
       failOnStatusCode: false,
     }).then((response) => {
-      expect(response.status).to.eq(401);
+      expect(response.status).to.eq(404);
     });
   });
-  it('Auth -> Connecting correctly with admin user.', () => {
+  it('Should fail when trying to connect without any data 1/2.', () => {
     cy.request({
       method: 'POST',
       url: '/api/auth/login',
-      body: {
-        username: 'admin@example.org',
-        password: 'admin',
-      },
       failOnStatusCode: false,
     }).then((response) => {
-      jwtUser = response.body.access_token;
-      idUserAdmin = response.body.user_id;
-      expect(response.status).to.eq(201);
+      expect(response.status).to.eq(401);
     });
   });
-  it('Auth -> Check self-profile of admin user with wrong jwt.', () => {
+  it('Should fail when trying to connect without any data 2/2.', () => {
     cy.request({
       method: 'GET',
-      url: `/api/users/${idUserAdmin}`,
+      url: '/api/auth/login',
       failOnStatusCode: false,
-      auth: {
-        bearer: 'incorrect_jwt',
-      },
     }).then((response) => {
-      expect(response.status).to.eq(403);
+      expect(response.status).to.eq(404);
     });
   });
-  it('Auth -> Check self-profile of admin user with correct jwt.', () => {
+  it('Connecting with incorrect user.', () => {
     cy.request({
-      method: 'GET',
-      url: `/api/users/${idUserAdmin}`,
-      failOnStatusCode: false,
-      auth: {
-        bearer: jwtUser,
+      method: 'POST',
+      url: '/api/auth/login',
+      body: {
+        email: faker.internet.email(),
+        password: faker.internet.password(),
       },
+      failOnStatusCode: false,
     }).then((response) => {
-      expect(response.status).to.eq(200);
-      expect(response.body.id).to.eq(idUserAdmin);
-      expect(response.body.role).to.eq('ADMIN');
-      expect(response.body.active).to.eq(true);
-      expect(response.body.email).to.eq('admin@example.org');
+      expect(response.status).to.eq(401);
     });
   });
-  it('Auth -> Connecting with proper editor user but wrong password.', () => {
+  it('Connecting with proper user admin but wrong admin password.', () => {
     cy.request({
       method: 'POST',
       url: '/api/auth/login',
       body: {
-        email: 'editor@example.org',
-        password: 'unknown_user_pw',
+        email: 'admin@example.org',
+        password: faker.internet.password(),
       },
       failOnStatusCode: false,
     }).then((response) => {
       expect(response.status).to.eq(401);
     });
   });
-  it('Auth -> Connecting correctly with editor user.', () => {
+  it('Login as admin', () => {
```

Comment:
```suggestion
  it('Should connect as admin', () => {
```

Be consistent with terms used `Login/Connecting`

Reply:
If I had to refactor this, I would rather use "login" instead of "connect".
This could be the subject of a new issue tho.

Reply:
https://github.com/datatlas-erasme/datatlas/issues/263